Exercise_A/index.html

```diff
@@ -10,8 +10,40 @@
       name="viewport"
       content="width=device-width, initial-scale=1"
     />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
   </head>
   <body>
     <!-- CODE HERE -->
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand" href="#">
+        <img src="/Exercise_A/assets/bootstrap.png" alt="Brand" style="height:50px">

```

Comment:
Great Job on the exercise, but to refer to the image you have to use ../Exercise_A/assets/bootstrap.png in order for the image to load.